### ISSUE TITLE
Crash on recovery phrase verification screen

### ISSUE THREAD

**Issue by kylehickinson** [brave/brave-ios#4938]
### Description: 

_Based on a crash report_:

[2022-01-31_08-16-36.7256_-0600-1d3a21039bf39517a135fdec90f19849b3b0fc9d.txt](https://github.com/brave/brave-ios/files/7998506/2022-01-31_08-16-36.7256_-0600-1d3a21039bf39517a135fdec90f19849b3b0fc9d.txt)

```
0   BraveWallet                   	0x0000000106d689d0 Swift runtime failure: Index out of range + 0 (<compiler-generated>:0)
1   BraveWallet                   	0x0000000106d689d0 specialized Array.remove(at:) + 152
2   BraveWallet                   	0x0000000106d689c4 specialized Array._makeMutableAndUnique() + 8 (<compiler-generated>:0)
3   BraveWallet                   	0x0000000106d689c4 specialized Array.remove(at:) + 140
4   BraveWallet                   	0x0000000106c6e380 closure #1 in SelectedWordsBox.tappedWord(atIndex:) + 28 (VerifyRecoveryPhraseView.swift:162)
5   BraveWallet                   	0x0000000106c6e380 partial apply for closure #1 in SelectedWordsBox.tappedWord(atIndex:) + 52 (<compiler-generated>:0)
6   BraveWallet                   	0x0000000106c70ebc thunk for @callee_guaranteed () -> (@error @owned Error) + 20 (<compiler-generated>:0)
7   BraveWallet                   	0x0000000106c6d510 partial apply for thunk for @callee_guaranteed () -> (@error @owned Error) + 16 (<compiler-generated>:0)
8   BraveWallet                   	0x0000000106c6e7d0 thunk for @callee_guaranteed () -> (@error @owned Error)partial apply + 12
9   SwiftUI                       	0x0000000188a8c5f8 closure #1 in withTransaction<A>(_:_:) + 92 (Transaction.swift:97)
10  SwiftUI                       	0x0000000188ac33b0 partial apply for closure #1 in withTransaction<A>(_:_:) + 24 (<compiler-generated>:0)
11  libswiftCore.dylib            	0x0000000185a76004 withExtendedLifetime<A, B>(_:_:) + 28 (LifetimeManager.swift:27)
12  SwiftUI                       	0x0000000188ab579c withTransaction<A>(_:_:) + 72 (Transaction.swift:88)
13  SwiftUI                       	0x0000000188aa65bc withAnimation<A>(_:_:) + 196 (Transaction.swift:112)
14  BraveWallet                   	0x0000000106c6de44 SelectedWordsBox.tappedWord(atIndex:) + 60 (VerifyRecoveryPhraseView.swift:161)
15  BraveWallet                   	0x0000000106c6de44 closure #1 in closure #1 in SelectedWordsBox.view(for:) + 60 (VerifyRecoveryPhraseView.swift:176)
16  BraveWallet                   	0x0000000106c6de44 partial apply for closure #1 in closure #1 in SelectedWordsBox.view(for:) + 88 (<compiler-generated>:0)
```

Looks like it's something to do with tapping to remove a selected word results in an index-out-of-range exception

**Comment by StephenHeaps** [comment]
Fixed in [PR #5220](https://github.com/brave/brave-ios/pull/5220) / [Issue #5219](https://github.com/brave/brave-ios/issues/5219)

**Comment by srirambv** [comment]
Verification passed on iPhone 7+ with iOS 14.8.1 running 1.38 (22.4.25.12)
- Verified steps from [#5220](https://github.com/brave/brave-ios/pull/5220)
- Verified no crash when double clicking/tapping the recovery phrase in the verification screen
------------
Verification passed on iPhoneXR with iOS 15.3.1 running 1.38 (22.4.25.12)
- Verified steps from [#5220](https://github.com/brave/brave-ios/pull/5220)
- Verified no crash when double clicking/tapping the recovery phrase in the verification screen
------------
Verification passed on iPad Pro with iOS 15.5 Beta 3 running 1.38 (22.4.25.12)
- Verified steps from [#5220](https://github.com/brave/brave-ios/pull/5220)
- Verified no crash when double clicking/tapping the recovery phrase in the verification screen
